exercises/clock/examples/success-standard/src/Clock.hs

```diff
@@ -21,4 +19,4 @@ clockMin :: Clock -> Int
 clockMin = (`mod` 60) . unClock
 
 toString :: Clock -> String
-toString c = printf "%02d:%02d" (clockHour c) (clockMin c)
+toString c = printf "%02d:%02d" (clockHour c) (clockMin c)
```

Comment:
I think the whitespace change, being unrelated to removing Num, should be amended away from this PR

(also I think the trailing newline should stay, but whether I think that is not really relevant to this PR)